content/pytorch/concepts/tensors/terms/cat/cat.md

```diff
@@ -0,0 +1,155 @@
+---
+Title: '.cat()'
+Description: 'Concatenates two or more tensors in the same dimension.'
+Subjects:
+  - 'AI'
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Deep Learning'
+  - 'Functions'
+  - 'Machine Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
+
+The **`.cat()`** function in PyTorch is used for concatenating two or more tensors in the same dimension. The tensors must have the same shape in all dimensions except the one in which they are concatenated.
+
+## Syntax
+
+```pseudo
+torch.cat(tensors, dim=0)
```

Comment:
Can we also mention the out parameter here? I think it would be helpful. Example:

torch.cat(tensors, dim=0, out=None)